server/src/main/java/org/elasticsearch/cluster/node/DiscoveryNodes.java

```diff
@@ -84,6 +84,16 @@ private DiscoveryNodes(
         return nodes.valuesIt();
     }
 
+    @Override
+    public int size() {
+        return nodes.size();
+    }
+
+    @Override
+    public Spliterator<DiscoveryNode> spliterator() {
+        return super.spliterator();
+    }
```

Comment:
Is it intentional?

Reply:
Good catch. It was used to find out all the usages of the spliterator, I will remove the override.